### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Vasyl-Poremchuk**

### CODE REVIEW

File: services/order.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+            )
+
+        return order
+
+
+def get_orders(username: str = None) -> QuerySet:
+    if username:
+        return Order.objects.filter(user__username=username)
+
+    return Order.objects.order_by("-user")
```
Comment (fsocie7y) [inline services/order.py:35]: i think u shouldn't do this) this was not in the condition

Comment (fsocie7y) [reply services/order.py:35]: just return queryset

File: services/order.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        tickets: list[dict[str, int]], username: str, date: str = None
+) -> Order:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        order = Order.objects.create(user=user)
+
+        if date:
+            order.created_at = date
+
+        order.save()
```
Comment (fsocie7y) [inline services/order.py:18]: i think it's "order.save()" should be inside "if" statment

**Review by fsocie7y — CHANGES_REQUESTED** [review]

**Review by fsocie7y — APPROVED** [review]
GGWP)

**Review by Y-Havryliv — APPROVED** [review]
Well done!

**Pull request closed without merging**